src/main/java/com/mixpanel/android/viewcrawler/ViewSnapshot.java

```diff
@@ -109,6 +109,8 @@ public void snapshots(UIThreadSet<Activity> liveActivities, OutputStream out) th
             if (MPConfig.DEBUG) {
                 Log.i(LOGTAG, "Screenshot took more than 1 second to be scheduled and executed. No screenshot will be sent.", e);
             }
+            writer.write("\"\"");
```

Comment:
technically, it doesn't matter. BUT, YOU ARE RIGHT! I should follow the same snapshot format as if the app is inactive.